### ISSUE TITLE
Freedoom IWADs launcher preview terribly out of date

### ISSUE THREAD

**Issue by Blastfrog** [team-eternity/eternity#561]
Freedoom1.wad, Freedoom2.wad and FreeDM.wad in the launcher menu all show terribly out of date titlepics. The title screen hasn't looked like that for several years.

**Comment by ioan-chera** [comment]
I wonder if this can be done smarter, by really taking the title pic from the found IWAD. Right now it takes the title pics from one of Eternity's own bundled wads, which are basically PNGs ripped from the IWADs.

**Comment by Altazimuth** [comment]
> I wonder if this can be done smarter, by really taking the title pic from the found IWAD.

Nope. Think about the Heretic shareware image. It's an edit, afaik.

**Comment by Catoptromancy** [comment]
Here are new ones. Scaled to size and 8bpp, same format as others. Includes one for Blasphemer, for future use.
https://www.geocities.ws/catodemos/doom/eternity/update_startupwad.zip
